scripts/get-last-rec-from-host

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
Can we source https://github.com/jcantrill/cluster-logging-tools/blob/master/scripts/env-prep and ref namespace so we ensure all the scripts are running against the correct namespace.  you may be able also to remove logic to fetching the espod